rdf2vec/converters.py

```diff
@@ -17,7 +27,19 @@ def create_kg(triples, label_predicates):
 
 
 def rdflib_to_kg(file, filetype=None, label_predicates=[]):
-    """Convert a rdflib.Graph (located at file) to our KnowledgeGraph."""
+    """Converts a rdflib.Graph to a knowledge graph.
+
+    Args:
+        file (file-like): The file that contains the rdflib.Graph
```

Comment:
it's a file_path, not a file (so we might have to change the name). So it's str as well and not a file-pointer.